src/test/scss/tools/mixins/type.spec.scss

```diff
@@ -0,0 +1,39 @@
+@use 'true' as *;
+@use 'src/scss/tools/mixins/type' as systemUnderTest;
+
+@include describe('rem [mixin]') {
+    @include it('outputs the expected CSS') {
+        // arrange
+        $property: padding;
+        $sizeValue: 10.5;
+
+        // act & assert
+        @include assert {
+            @include output {
+                @include systemUnderTest.rem($property, $sizeValue);
+            }
+
+            @include expect {
+                padding: 11px;
```

Comment:
Is it expected to round 10.5 to 11? Or is that what you're testing? (If so, might be worth adding it to the test description)

Reply:
Yeah that's it - I'll rename the test

Reply:
Yeah that's part of what's tested - will update the test description!